nomt/Cargo.toml

```diff
@@ -29,6 +29,7 @@ lru = "0.12.3"
 libc = "0.2.155"
 criterion = { version = "0.3", optional = true }
 thread_local = "1.1.8"
+fs2 = "0.4.3"
```

Comment:
This is not my favorite, as this doesn't give us much. We don't care about supporting much besides modern Linux and macOS, we don't explicitly target Windows and can just directly use `flock`. But I thought for now this is OK and we should reconsider that in #501.